### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by dscardaci**
Removed placeholder banner

# Summary 

----

**Related issue :**

### CODE REVIEW

**Review by andrea-manzi — APPROVED** [review]

File: content/en/providers/_index.md [hunk 1]
```diff
@@ -8,7 +8,4 @@ menu:
     weight: 20
 ---
```
Comment (gwarf) [inline content/en/providers/_index.md:10]:
```suggestion

This section contains documentation aiming at Service Providers.
See [the users section](../users) for User-centric documentation.
```

File: content/en/providers/_index.md [hunk 2]
```diff
@@ -8,7 +8,4 @@ menu:
...
 
-{{% pageinfo %}}
-This is a placeholder page for Service providers documentation.
-{{% /pageinfo %}}
```
Comment (gwarf) [inline content/en/providers/_index.md:13]: LGTM, still we need at lease a dummy sentence on the page

**Review by gwarf — COMMENTED** [review]
I will add a dummy sentence before merging this.

**Review by gwarf — APPROVED** [review]
LGTM with the "small intro"

**Pull request merged**